Packs/SecureWorks/ReleaseNotes/4_1_0.md

```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+##### TaegisXDR
+- Added the following new commands:
+    - ***taegisxdr-create-comment***
+    - ***taegisxdr-fetch-comment***
+    - ***taegisxdr-fetch-comments***
+    - ***taegisxdr-update-comment***
+- Added ***url*** field output for reference in most commands.
```

Comment:
Where is this done in the code?

Reply:
Sure thing. If you're referring to the `url` field, an example of this is can be seen, on line Line 875 of TaegixXDR.py. This is to work with the `url_keys` of the `tableToMarkdown` method. 

I did this everywhere it made sense. If needed, I can remove this from the release notes.

Reply:
@moishce Does this makes sense? I'd like to get this PR completed as soon as possible as we have a few users awaiting these changes.

Reply:
I think it can be removed from the release notes
It can be confusing

Reply:
Sure thing. I've removed it from the release notes.